dev-support/mini-submarine/conf/submarine.xml

```diff
@@ -19,7 +19,7 @@
 <configuration>
   <property>
     <name>submarine.runtime.class</name>
-    <value>org.apache.hadoop.yarn.submarine.runtimes.tony.TonyRuntimeFactory</value>
+    <value>org.apache.submarine.runtimes.tony.TonyRuntimeFactory</value>
     <!--
     <value>org.apache.hadoop.yarn.submarine.runtimes.yarnservice.YarnServiceRuntimeFactory</value>
     -->
```

Comment:
we also need to change it to **org.apache.submarine.runtimes.yarnservice.YarnServiceRuntimeFactory**

Reply:
It makes sense to me.
Thanks, @pingsutw